src/borg/archiver/_common.py

```diff
@@ -249,9 +249,14 @@ def wrapper(self, args, repository, manifest, **kwargs):
 class Highlander(argparse.Action):
     """make sure some option is only given once"""
 
+    def __init__(self, *args, **kwargs):
+        self.called = False
+        super(Highlander, self).__init__(*args, **kwargs)
```

Comment:
You can just use `super().__init__(*args, **kwargs)`. The more verbose way was just needed for old pythons.

There is some risk that the super class also uses `.called`, so maybe we rather better use `.__called`?